projects/rero/ng-core/src/lib/record/search/aggregation/buckets/buckets.component.scss

```diff
@@ -0,0 +1,55 @@
+/*
+ * RERO angular core
+ * Copyright (C) 2020 RERO
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU Affero General Public License as published by
+ * the Free Software Foundation, version 3 of the License.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * GNU Affero General Public License for more details.
+ *
+ * You should have received a copy of the GNU Affero General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+@import "node_modules/bootstrap/scss/functions";
+@import "node_modules/bootstrap/scss/mixins";
+@import "node_modules/bootstrap/scss/variables";
+
+.bucket-group {
+
+  .bucket-item {
+    font-size: 0.9rem;
+
+    .bucket-data {
+
+      .form-check-label {
+        color: $body-color;
+        padding: 0 0.5rem 0 0.3rem;
```

Comment:
I would be great to use Bootstrap variables or css functions.

Reply:
done